vassal-app/src/main/java/VASSAL/build/module/GameState.java

```diff
@@ -435,6 +435,30 @@ protected Command myUndoCommand() {
     }
   }
 
+  public static final int NO_NEED_TO_SAVE = -2;
+
+  /**
+   * Offers player the chance to save the game if an unsaved one is
```

Comment:
is what?

Reply:
An extra status for the thing to return (in addition to the ones from the dialog box itself)

Reply:
I mean, it's just not a complete thought. Aren't some words missing?

Reply:
Oh right, fixed